### PR TITLE
Unconscious and Knockdown are now status effects

### LINKED ISSUE #28702: revhead flash stun times
[Note]: #28670: 'Makes oldchat show while goonchat loads, sends preload messages there.' by MrStonedOne at commit 7a1393
Based off origin/master commit: 58c054

Revheads flashing someone will make them go down for a really long time, more so than a normal person flashing someone.

### PR DESCRIPTION
**Pull request by ChangelingRain**
**4/4 Complete**
Fixes #28702

### CODE REVIEW

**Comment by optimumtact** [comment]
150 hours

File: code/datums/status_effects/status_effect.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	var/alert_type = /obj/screen/alert/status_effect //the alert thrown by the status effect, contains name and description
 	var/obj/screen/alert/status_effect/linked_alert = null //the alert itself, if it exists
 
-/datum/status_effect/New(mob/living/new_owner)
+/datum/status_effect/New(mob/living/new_owner, terrifying_second_argument)
```
Comment (ChangelingRain) [inline code/datums/status_effects/status_effect.dm]: This is very bad but I couldn't figure out how to do arbitrary arguments like Initialize() does.

Comment (Cyberboss) [inline code/datums/status_effects/status_effect.dm]: Arg forwarding:

/proc/succthiccdiccquicc(peanut, almonds)
   return
/proc/des(nuts, ...)
   var/arg1 = "peanuts"
   succthiccdiccquicc(arglist(list(arg1) + args))
```

Comment (ChangelingRain) [inline code/datums/status_effects/status_effect.dm]: As in, I did try it and it didn't work;
```
/datum/status_effect/New(list/arguments)
    on_creation(arglist(arguments))

/datum/status_effect/proc/on_creation(mob/living/new_owner, ...)
	if(new_owner)
		owner = new_owner
	if(owner)
		LAZYADD(owner.status_effects, src)
	addtimer(CALLBACK(src, .proc/start_ticking), 0) //Give us the minimum possible time to set any variables

/mob/living/proc/apply_status_effect(effect, ...) //applies a given status effect to this mob, returning the effect if it was successful
	. = FALSE
	var/datum/status_effect/S1 = effect
	LAZYINITLIST(status_effects)
	for(var/datum/status_effect/S in status_effects)
		if(S.id == initial(S1.id) && S.status_type)
			if(S.status_type == STATUS_EFFECT_REPLACE)
				S.be_replaced()
			else
				return
	args[1] = src
	S1 = new effect(args)
	. = S1
```

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -62,8 +62,9 @@
 
 /datum/emote/living/collapse/run_emote(mob/user, params)
```
Comment (lzimann) [inline code/modules/mob/living/emote.dm]: Note to self, typecast all these `mob/user` to living one day.

**Review by lzimann — APPROVED** [review]
Everything looks good.

**Pull request merged**